Content.Shared/Throwing/ThrowingSystem.cs

```diff
@@ -228,5 +231,8 @@ public void TryThrow(EntityUid uid,
             if (!msg.Cancelled)
                 _physics.ApplyLinearImpulse(user.Value, -impulseVector / physics.Mass * pushbackRatio * MathF.Min(massLimit, physics.Mass), body: userPhysics);
         }
+
+        var popup = Loc.GetString("throwing-user-threw-others", ("user", user), ("thrown", uid));
```

Comment:
need to use Identity.Entity() in `Content.Shared.IdentityManagement` for the user and uid so that it respects all the junk correctly.